js/app/modules/contentGroup/contentGroup.js

```diff
@@ -114,7 +137,23 @@ const ContentGroup = new Module.Class({
             no_show_all: true,
             vexpand: true,
         });
-        this._stack.add_named(this._arrangement, CONTENT_PAGE_NAME);
+
+        let see_more_button = new Gtk.Button({
+            halign: Gtk.Align.CENTER,
+            label: 'See more', // Gettext this
```

Comment:
Hmm, I'd like to minimize the customizable surface area as much as possible. Unlike with the trigger, I don't see a reason why this would need to say anything other than "See more"